repository-data/application/src/main/resources/hcm-content/content/documents/administration/configuration/local-feature---visitscotland.com.yaml

```
@@ -0,0 +1,202 @@
+      links.internal-sites, links.convert-to-relative, og.site-name, og.twitter.site,
+      seo.alternate-link-locale-order, seo.title-suffix, seo.og.site-name, seo.og.twitter.site,
+      gtm.is-production, gtm.container-id, gtm.preview-query-string, form.is-production,
+      global-search.path, global-search.engine-id, ski.timeout, site.path.global-search,
+      site.path.ski-landing, site.path.about-us, site.path.icentre-landing, site.path.newsletter,
+      site.path.campaigns, iknow-module.enabled, form.json.countries, form.json.messages,
+      form.marketo.munchkin, form.maketo.instance-url, form.recaptcha-key, form.marketo.scripts,
+      form.marketo.script, links.download.extensions, events-listings.page-size, form.breg.legal-basis.enable,
+      site.path.banner, cludo.customer-id, cludo.engine-id, cludo.experience-id, feature.products-search.enable,
+      feature.table-of-contents.enable, feature.global-search.enable, form.breg.legal-basis.text]
+    resourcebundle:messages: ['https://community.visitscotland.com/', discussions/tagged/,
+      'localhost,www.visitscotland.com,beta.visitscotland.com,blog.visitscotland.com',
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 30 but found 6

(indentation)